sources/packages/web/src/views/PersonalInfoQuestionnaire.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="p-component">
+  <div class="p-component p-m-8">
```

Comment:
Actually am unsure about adding m-* classes simply because we are not testing for them are we?
My thought is they are for mobile devices or smaller screens. Even i have been adding them unconvincingly. can you initiate a chat/discussion around this point please

Reply:
This is size class and for spacing only.